e2e/test/scenarios/admin-2/whitelabel.cy.spec.js

```diff
@@ -292,17 +293,25 @@ describeEE("formatting > whitelabel", () => {
     });
 
     it("should allow users to provide internal urls", () => {
-      cy.findByTestId("landing-page").click().clear().type("/test-1").blur();
-      cy.wait(["@putLandingPage", "@getSettings"]);
+      cy.findByLabelText("Landing page custom destination")
+        .click()
+        .clear()
+        .type("/test-1")
+        .blur();
+      undoToast().findByText("Saved").should("be.visible");
```

Comment:
We use the label `Changed saved` in master, but not in 49, so I need to modify the test to reflect that.